duckduckgo/iOS
# Prepare app for localization

Task/Issue URL: https://app.asana.com/0/856498667320406/1176971145104443
Tech Design URL:
CC:

**Description**:
Prepare for localization work:
- Go through UserText and provide comments when needed.
- Ensure special characters (like %) are included in text instead of being concatenated.
- Switch to plural stiringsdict where needed.
- Extract attributed labels from storyboards to UserText and apply these from the code.
- Regenerate Localizable.strings from the code.

To resolve later:
- About screen.

**Steps to test this PR**:

Smoke tests strings in the app, especially:
- Dax dialogs.
- Privacy protection trackers summary.

To ensure strings are translatable you can choose 'Accented Pseudolanguage' in the scheme while running the app.



**Copy Testing**:


**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
